cmd/config/api/help.go

```diff
@@ -39,5 +39,11 @@ var (
 			Optional:    true,
 			Type:        "csv",
 		},
+		config.HelpKV{
+			Key:         apiRemoteTransportDeadline,
+			Description: `set the deadline for API requests on remote transports while proxying between federated instances e.g. "2h"`,
```

Comment:
I think 'remote transports' keywords are not clear for users.

```
set the deadline for API requests while proxying between federated instances
```

Reply:
It is technically correct, since the usage of this is rare and niche - I think we are okay to call it this way..